### ISSUE TITLE
Overview cheat sheet

### ISSUE THREAD

**Issue by soerenthomsen** [OceanGlidersCommunity/Oxygen_SOP#161]
@pipa85 suggested to have a simple overview graph to understand key steps in 10 min. 

I agree with her that this would be great to have. @patricialg we started with that once at the beginning and postponed due to too many changes. But I think we should include this before submission or?

**Comment by patricialg** [comment]
Yes, I still have it ;) but unfinished. I'll try create a draft so we can comment and find the best option for the diagram.

**Comment by soerenthomsen** [comment]
Great!

**Comment by patricialg** [comment]
![Picture1](https://user-images.githubusercontent.com/13015381/142466828-bafbe37d-20af-4510-8d0e-f7c0519b6574.png)
This is what we started a few months ago, where processes are calssified as:
_!!! Must have
!! Should have
! Desirable_ 
 I will need to check for updates. 
Is this a good design? Or we prefer a cascade?

**Comment by pipa85** [comment]
This looks good. although a simple 'shopping' list would also work. it would also allow to have a sentence or so below with details if needed.  I would mention if you don't have an oxygen sensor on the rosette, but can collect samples for Winkler titrations, that should be done!

**Comment by soerenthomsen** [comment]
Agree Winkler Samples if attached to CTD should be added!

**Comment by soerenthomsen** [comment]
@hahn-johannes what is your view on this? What are "key" steps to be taken?

**Comment by tomhull** [comment]
For me a glider oxygen shopping list would look like:

1. mount the sensors as best you can, buy ones which are better for gliders (imo that means, RBR / RINKO)
2. make sure your glider is configured to record phase with correct timings
3. two point calibrate sensor just prior to use
4. have some form of in-situ reference. we have a hierarchy of decreasing quality, e.g.
    - BEST: multiple co-located CTD casts in well mixed waters with lots of winklers + in-air drift correction
    - GOOD: single point winkler sample and CTD from nearby and no drift correction
    - OK: in-air calibration only
5. correct for optode lag

**Comment by soerenthomsen** [comment]
I add to "Best" that optode is attached to calibrated CTD and see the co-located CTD casts more as a "check"

**Comment by patricialg** [comment]
OK, so following what we have in the SOP now and your comments, we can have something like this? (we will need to improve it once we decide the final version, and I'll try to add it to a diagram that looks better ;) )

**Pre-deployment**
1.	Check that the sensors are in good condition.
2.	Do two-point calibration. Ideallly, test more than one sensor.
3.	Mount the sensors as best you can, buy ones which are better for gliders (imo that means, RBR / RINKO)
4.	Have some form of in-situ reference.
- OK: In situ intercomparison in the tank during ballasting
5.	Configure sensor for deployment.

**Deployment** 
1.	Make sure your glider is configured to record phase with correct timings.
2.	Keep sensor foil wet.
3.	Do two-point calibration prior to use
4.	Have some form of in-situ reference. The hierarchy of decreasing quality, e.g. 
- BEST: Optodes attached to a CTD. This is for example possible via a logger for the Aanderaa ones and see the co-located CTD casts
- BEST: multiple co-located CTD casts in well mixed waters with lots of Winkler samples and in-air drift correction
- GOOD: single point winkler sample and CTD from nearby and no drift correction
- OK: in-air calibration only

**Recovery**
1.	After recovery, keep sensor foil wet.
2.	Download data.
3.	Have some form of in-situ reference. The hierarchy of decreasing quality, e.g. 
 - BEST: Optodes attached to a CTD. This is for example possible via a logger for the Aanderaa ones and see the co-located CTD casts
- BEST: multiple co-located CTD casts in well mixed waters with lots of Winkler samples and in-air drift correction
- GOOD: single point Winkler sample and CTD from nearby and no drift correction
- OK: in-air calibration only
4.	Properly clean and store the sensor.
5.	Correct for optode lag

**Comment by soerenthomsen** [comment]
@gkrahmann we plan to close the community review for the oxygen SOP this week to prepare a submission to OBPS. What do you think about the "cheat sheet/list" above?

**Comment by gkrahmann** [comment]
A bit in line with what I suggested a long time ago, I would separate the
'recovery'  into
'recovery'  and  'post-processing'    or 'dmqc' as in the sections.
Then 'post-processing' would include
1. Determine optode lag           <- this one depends on optode type and mounting
2. Correct optode lag 

I also would be cautious about the   'imo'  from Tom. Either that is an agreed upon statement,
or leave it out...
I guess too few of us have experience with different optodes to make a qualified statement.
And if you have statement about buying sensors, it should perhaps be the first one ;-)

**Comment by soerenthomsen** [comment]
Thanks Gerd! I agree with all your points. Also to include the other chapters and have bullit point style key steps to do. Everything together should not be then a page. 

Also which sensor to buy I think should be remove here.

**Comment by patricialg** [comment]
Hi all,
Following your recomendations:

**Pre-deployment/Deployment**
1.	Check that the sensors are in good condition and you have selected the best option for the planned mission.
2.	BEST: Do two-point calibration. Ideally, test more than one sensor.
3.	Mount the sensors
4.	OK: In situ intercomparison in the tank during ballasting as in-situ reference.
5.	Configure sensor for deployment. Make sure your glider is configured to record phase with correct timings.
6.	Keep sensor foil wet at least 8 hours before deployment. If it cannot be kept submerged in water, have a wet sponge covering the foil.
7.	Do two-point calibration prior to use
8.	In-situ reference measurements recommended to do, using hierarchy of decreasing quality: 
•	BEST: Optodes attached to a CTD. 
•	BEST: multiple co-located CTD casts in well mixed waters (including Winkler samples at different depths) and in-air drift correction
•	GOOD: single point Winkler sample and CTD from nearby and no drift correction
•	OK: in-air calibration only
**Recovery**
1.	Keep sensor foil wet.
2.	Download data.
3.	In-situ reference measurements recommended to do, using hierarchy of decreasing quality: 
•	BEST: Optodes attached to a CTD. 
•	BEST: multiple co-located CTD casts in well mixed waters (including Winkler samples at different depths) and in-air drift correction
•	GOOD: single point Winkler sample and CTD from nearby and no drift correction
•	OK: in-air calibration only
4.	Clean and store the sensor.
**DMQC**
1.	Determine optode lag 
2.	Correct optode lag

Let me know what you think we should include/delete. 
Cheers,
Patry

**Comment by soerenthomsen** [comment]
I close this discussion for now. My suggestion is to start working on a graphical version after version 1.0 on OBPS i.e. during the preparation for publication for Frontiers.
